src/txn/txn_rollback_to_stable.c

```diff
@@ -232,8 +232,10 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
          * we can skip it.
          */
         if (__wt_txn_visible_all(
-              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts))
+              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts)) {
+            WT_STAT_CONN_INCR(session, cursor_prev_hs_tombstone);
```

Comment:
I think it'd be useful to split the rollback to stable counters into a different statistic.

Reply:
Yeah I thought about that but it raised the question of how many statistics we'd need to differentiate well, hopefully two will be enough.